core/src/main/resources/config.yml

```diff
@@ -115,6 +115,12 @@ economy:
   # Used to replace %currency% for the language files and the 2nd hologram line.
   server-currency: $
   use-xp: false
+  #taxes in %. Leave 0 if you don't want taxes. Taxes when a player buy from a chest.
+  taxes-shop-player: 15
```

Comment:
Set taxes to 0 (disabled) by default - it's a nice feature, but not every server wants it and it's less confusing if your money doesn't magically vanish.